.editorconfig

```diff
@@ -2,3 +2,13 @@ root = true
 
 [*]
 charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+indent_size = 4
```

Comment:
```suggestion
```
Since tab widths are not explicitly defined in a file but are purely visual – it's always 1 character per each indentation level compared to space indentation with 2, 4, or whatever many hard-coded space characters – it's practical to leave it to the user for customization.